OMP: Info #212: KMP_AFFINITY: decoding x2APIC ids.
OMP: Info #213: KMP_AFFINITY: cpuid leaf 11 not supported - decoding legacy APIC ids.
OMP: Info #149: KMP_AFFINITY: Affinity capable, using global cpuid info
OMP: Info #154: KMP_AFFINITY: Initial OS proc set respected: 0,1
OMP: Info #156: KMP_AFFINITY: 2 available OS procs
OMP: Info #157: KMP_AFFINITY: Uniform topology
OMP: Info #159: KMP_AFFINITY: 2 packages x 1 cores/pkg x 1 threads/core (2 total cores)
OMP: Info #214: KMP_AFFINITY: OS proc to physical thread map:
OMP: Info #171: KMP_AFFINITY: OS proc 0 maps to package 0 
OMP: Info #171: KMP_AFFINITY: OS proc 1 maps to package 1 
OMP: Info #250: KMP_AFFINITY: pid 13708 tid 13708 thread 0 bound to OS proc set 0
INFO:tensorflow:Using default config.
WARNING:tensorflow:Using temporary folder as model directory: /tmp/tmppu_t5_j7
INFO:tensorflow:Using config: {'_model_dir': '/tmp/tmppu_t5_j7', '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_steps': None, '_save_checkpoints_secs': 600, '_session_config': allow_soft_placement: true
graph_options {
  rewrite_options {
    meta_optimizer_iterations: ONE
  }
}
, '_keep_checkpoint_max': 5, '_keep_checkpoint_every_n_hours': 10000, '_log_step_count_steps': 100, '_train_distribute': None, '_device_fn': None, '_protocol': None, '_eval_distribute': None, '_experimental_distribute': None, '_service': None, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x7fe2e7e41cc0>, '_task_type': 'worker', '_task_id': 0, '_global_id_in_cluster': 0, '_master': '', '_evaluation_master': '', '_is_chief': True, '_num_ps_replicas': 0, '_num_worker_replicas': 1}
WARNING:tensorflow:From /home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From /home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/inputs/queues/feeding_queue_runner.py:62: QueueRunner.__init__ (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
WARNING:tensorflow:From /home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/inputs/queues/feeding_functions.py:500: add_queue_runner (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
INFO:tensorflow:Calling model_fn.
WARNING:tensorflow:From /home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow/python/keras/layers/core.py:143: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
WARNING:tensorflow:From /home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow/python/ops/array_grad.py:425: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
/home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow/python/ops/gradients_impl.py:110: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Graph was finalized.
2019-06-11 19:46:59.666538: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2100040000 Hz
2019-06-11 19:46:59.666870: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x561113ccb5a0 executing computations on platform Host. Devices:
2019-06-11 19:46:59.666904: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
2019-06-11 19:46:59.667045: I tensorflow/core/common_runtime/process_util.cc:71] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:From /home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/monitored_session.py:809: start_queue_runners (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 0 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
OMP: Info #250: KMP_AFFINITY: pid 13708 tid 13724 thread 1 bound to OS proc set 1
OMP: Info #250: KMP_AFFINITY: pid 13708 tid 13758 thread 2 bound to OS proc set 0
OMP: Info #250: KMP_AFFINITY: pid 13708 tid 13725 thread 3 bound to OS proc set 1
OMP: Info #250: KMP_AFFINITY: pid 13708 tid 13759 thread 4 bound to OS proc set 0
INFO:tensorflow:loss = 2.3000085, step = 1
INFO:tensorflow:global_step/sec: 0.976916
INFO:tensorflow:loss = 2.0089555, step = 101 (102.364 sec)
INFO:tensorflow:global_step/sec: 0.990832
INFO:tensorflow:loss = 1.8482678, step = 201 (100.925 sec)
INFO:tensorflow:global_step/sec: 0.992226
INFO:tensorflow:loss = 1.7418286, step = 301 (100.783 sec)
INFO:tensorflow:global_step/sec: 0.992262
INFO:tensorflow:loss = 1.7165713, step = 401 (100.780 sec)
INFO:tensorflow:Saving checkpoints for 468 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6804085.
INFO:tensorflow:Calling model_fn.
WARNING:tensorflow:From /home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow/python/ops/metrics_impl.py:455: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T18:55:41Z
INFO:tensorflow:Graph was finalized.
WARNING:tensorflow:From /home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/saver.py:1266: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-468
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-18:55:42
INFO:tensorflow:Saving dict for global step 468: accuracy = 0.8541, global_step = 468, loss = 1.6689014
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 468: /tmp/tmppu_t5_j7/model.ckpt-468
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-468
WARNING:tensorflow:From /home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/saver.py:1070: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 468 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6914209, step = 469
INFO:tensorflow:global_step/sec: 0.990153
INFO:tensorflow:loss = 1.6538911, step = 569 (100.995 sec)
INFO:tensorflow:global_step/sec: 0.999513
INFO:tensorflow:loss = 1.6774693, step = 669 (100.048 sec)
INFO:tensorflow:global_step/sec: 1.00146
INFO:tensorflow:loss = 1.6315639, step = 769 (99.854 sec)
INFO:tensorflow:global_step/sec: 1.00265
INFO:tensorflow:loss = 1.5967791, step = 869 (99.737 sec)
INFO:tensorflow:Saving checkpoints for 936 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6201271.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T19:05:05Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-936
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-19:05:06
INFO:tensorflow:Saving dict for global step 936: accuracy = 0.8836, global_step = 936, loss = 1.6102722
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 936: /tmp/tmppu_t5_j7/model.ckpt-936
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-936
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 936 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.588509, step = 937
INFO:tensorflow:global_step/sec: 0.987897
INFO:tensorflow:loss = 1.6263318, step = 1037 (101.226 sec)
INFO:tensorflow:global_step/sec: 0.997835
INFO:tensorflow:loss = 1.5350177, step = 1137 (100.217 sec)
INFO:tensorflow:global_step/sec: 1.00171
INFO:tensorflow:loss = 1.6248735, step = 1237 (99.829 sec)
INFO:tensorflow:global_step/sec: 0.999786
INFO:tensorflow:loss = 1.5879879, step = 1337 (100.021 sec)
INFO:tensorflow:Saving checkpoints for 1404 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.5859318.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T19:15:16Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-1404
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-19:15:17
INFO:tensorflow:Saving dict for global step 1404: accuracy = 0.8945, global_step = 1404, loss = 1.5907598
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 1404: /tmp/tmppu_t5_j7/model.ckpt-1404
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-1404
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 1404 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6109362, step = 1405
INFO:tensorflow:global_step/sec: 0.989533
INFO:tensorflow:loss = 1.5731242, step = 1505 (101.058 sec)
INFO:tensorflow:global_step/sec: 1.00132
INFO:tensorflow:loss = 1.6041605, step = 1605 (99.868 sec)
INFO:tensorflow:global_step/sec: 1.00102
INFO:tensorflow:loss = 1.6106157, step = 1705 (99.898 sec)
INFO:tensorflow:global_step/sec: 0.997079
INFO:tensorflow:loss = 1.5490657, step = 1805 (100.293 sec)
INFO:tensorflow:Saving checkpoints for 1872 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6371515.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T19:26:11Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-1872
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-19:26:13
INFO:tensorflow:Saving dict for global step 1872: accuracy = 0.8987, global_step = 1872, loss = 1.5795733
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 1872: /tmp/tmppu_t5_j7/model.ckpt-1872
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-1872
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 1872 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6375681, step = 1873
INFO:tensorflow:global_step/sec: 0.990452
INFO:tensorflow:loss = 1.5552589, step = 1973 (100.965 sec)
INFO:tensorflow:global_step/sec: 0.999035
INFO:tensorflow:loss = 1.5919423, step = 2073 (100.097 sec)
INFO:tensorflow:global_step/sec: 0.999432
INFO:tensorflow:loss = 1.5620888, step = 2173 (100.057 sec)
INFO:tensorflow:global_step/sec: 1.00031
INFO:tensorflow:loss = 1.5820208, step = 2273 (99.969 sec)
INFO:tensorflow:Saving checkpoints for 2340 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:From /home/admin/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/saver.py:966: remove_checkpoint (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to delete files with this prefix.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.5409756.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T19:37:52Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-2340
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-19:37:53
INFO:tensorflow:Saving dict for global step 2340: accuracy = 0.9019, global_step = 2340, loss = 1.572893
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 2340: /tmp/tmppu_t5_j7/model.ckpt-2340
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-2340
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 2340 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.5629246, step = 2341
INFO:tensorflow:global_step/sec: 0.990767
INFO:tensorflow:loss = 1.5724925, step = 2441 (100.933 sec)
INFO:tensorflow:global_step/sec: 0.998436
INFO:tensorflow:loss = 1.5811262, step = 2541 (100.157 sec)
INFO:tensorflow:global_step/sec: 0.977474
INFO:tensorflow:loss = 1.535008, step = 2641 (102.304 sec)
INFO:tensorflow:global_step/sec: 0.9976
INFO:tensorflow:loss = 1.6026365, step = 2741 (100.241 sec)
INFO:tensorflow:Saving checkpoints for 2808 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.5191115.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T19:50:21Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-2808
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-19:50:22
INFO:tensorflow:Saving dict for global step 2808: accuracy = 0.9053, global_step = 2808, loss = 1.5684294
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 2808: /tmp/tmppu_t5_j7/model.ckpt-2808
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-2808
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 2808 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.5893717, step = 2809
INFO:tensorflow:global_step/sec: 0.99219
INFO:tensorflow:loss = 1.5817077, step = 2909 (100.788 sec)
INFO:tensorflow:global_step/sec: 1.00125
INFO:tensorflow:loss = 1.5754085, step = 3009 (99.875 sec)
INFO:tensorflow:global_step/sec: 0.999866
INFO:tensorflow:loss = 1.5515745, step = 3109 (100.013 sec)
INFO:tensorflow:global_step/sec: 0.999606
INFO:tensorflow:loss = 1.5241055, step = 3209 (100.039 sec)
INFO:tensorflow:Saving checkpoints for 3276 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.5534263.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T20:03:34Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-3276
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-20:03:36
INFO:tensorflow:Saving dict for global step 3276: accuracy = 0.9078, global_step = 3276, loss = 1.5638355
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 3276: /tmp/tmppu_t5_j7/model.ckpt-3276
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-3276
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 3276 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.5653689, step = 3277
INFO:tensorflow:global_step/sec: 0.988852
INFO:tensorflow:loss = 1.5504327, step = 3377 (101.128 sec)
INFO:tensorflow:global_step/sec: 0.996142
INFO:tensorflow:loss = 1.5822151, step = 3477 (100.387 sec)
INFO:tensorflow:global_step/sec: 0.998161
INFO:tensorflow:loss = 1.5853859, step = 3577 (100.184 sec)
INFO:tensorflow:global_step/sec: 0.997853
INFO:tensorflow:loss = 1.5668812, step = 3677 (100.215 sec)
INFO:tensorflow:Saving checkpoints for 3744 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.587312.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T20:17:32Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-3744
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-20:17:33
INFO:tensorflow:Saving dict for global step 3744: accuracy = 0.9091, global_step = 3744, loss = 1.5614709
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 3744: /tmp/tmppu_t5_j7/model.ckpt-3744
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-3744
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 3744 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.5627468, step = 3745
INFO:tensorflow:global_step/sec: 0.991883
INFO:tensorflow:loss = 1.5523368, step = 3845 (100.819 sec)
INFO:tensorflow:global_step/sec: 1.00028
INFO:tensorflow:loss = 1.5619494, step = 3945 (99.973 sec)
INFO:tensorflow:global_step/sec: 1.00008
INFO:tensorflow:loss = 1.5896488, step = 4045 (99.991 sec)
INFO:tensorflow:global_step/sec: 1.00035
INFO:tensorflow:loss = 1.5246361, step = 4145 (99.965 sec)
INFO:tensorflow:Saving checkpoints for 4212 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.5613675.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T20:32:18Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-4212
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-20:32:19
INFO:tensorflow:Saving dict for global step 4212: accuracy = 0.9127, global_step = 4212, loss = 1.5584577
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 4212: /tmp/tmppu_t5_j7/model.ckpt-4212
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-4212
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 4212 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.5743155, step = 4213
INFO:tensorflow:global_step/sec: 0.98973
INFO:tensorflow:loss = 1.5585788, step = 4313 (101.038 sec)
INFO:tensorflow:global_step/sec: 0.999645
INFO:tensorflow:loss = 1.5499122, step = 4413 (100.036 sec)
INFO:tensorflow:global_step/sec: 0.999122
INFO:tensorflow:loss = 1.58484, step = 4513 (100.088 sec)
INFO:tensorflow:global_step/sec: 0.997363
INFO:tensorflow:loss = 1.5658315, step = 4613 (100.264 sec)
INFO:tensorflow:Saving checkpoints for 4680 into /tmp/tmppu_t5_j7/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.5747912.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T20:47:49Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmppu_t5_j7/model.ckpt-4680
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-20:47:50
INFO:tensorflow:Saving dict for global step 4680: accuracy = 0.9134, global_step = 4680, loss = 1.5575314
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 4680: /tmp/tmppu_t5_j7/model.ckpt-4680

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

For delta=1e-5, the current epsilon is: 0.91
Test accuracy after 1 epochs is: 0.854
For delta=1e-5, the current epsilon is: 0.93
Test accuracy after 2 epochs is: 0.884
For delta=1e-5, the current epsilon is: 0.95
Test accuracy after 3 epochs is: 0.895
For delta=1e-5, the current epsilon is: 0.97
Test accuracy after 4 epochs is: 0.899
For delta=1e-5, the current epsilon is: 0.99
Test accuracy after 5 epochs is: 0.902
For delta=1e-5, the current epsilon is: 1.01
Test accuracy after 6 epochs is: 0.905
For delta=1e-5, the current epsilon is: 1.03
Test accuracy after 7 epochs is: 0.908
For delta=1e-5, the current epsilon is: 1.05
Test accuracy after 8 epochs is: 0.909
For delta=1e-5, the current epsilon is: 1.07
Test accuracy after 9 epochs is: 0.913
For delta=1e-5, the current epsilon is: 1.09
Test accuracy after 10 epochs is: 0.913
Elapsed time:  7252.109696865082
